charts/opentelemetry-operator/values.yaml

```diff
@@ -90,6 +90,13 @@ admissionWebhooks:
   failurePolicy: Fail
   enabled: true
 
+  ## Customize webhook timeout duration
+  timeoutSeconds: 10
+
+  ## Provide selectors for your objects
+  # namespaceSelector: {}
```

Comment:
@Sadzeih pls uncomment these 2 empty values and provide commented examples of both.